chain/rust/src/crypto/utils.rs

```diff
@@ -0,0 +1,31 @@
+//impl BootstrapWitness {
```

Comment:
Is this meant to all be commented out?

Reply:
We haven't migrated the Byron code over yet which this depends on.